src/api/spec/models/project_spec.rb

```diff
@@ -33,7 +33,7 @@
             </project>
           EOF
         )
-        project.update_repositories(xml_hash, force = false)
+        project.update_repositories(xml_hash, false)
```

Comment:
IMHO we loose some context here by this change .... maybe we should turn this to a hash to stay readable?

{force: false}